### PR TITLE
Add Ingress resource in hokusai/staging.yml and hokusai/production.yml

### PR DESCRIPTION
**Pull request by izakp**
Addresses https://artsyproduct.atlassian.net/browse/PLATFORM-2583

Automated with https://github.com/artsy/ops-util/pull/6

### CODE REVIEW

**Comment by izakp** [comment]
@joeyAghion are you point person on this repo?  Assigning it to you.

**Comment by joeyAghion** [comment]
Sweet. The load-balancer deletion is a separate step, yeah?

File: hokusai/staging.yml [hunk 1]
```diff
@@ -108,3 +108,43 @@ spec:
...
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: artsy-wwwify
+  annotations:
+    nginx.ingress.kubernetes.io/whitelist-source-range: "{{ cloudflareIpSourceRanges|join(',') }}"
+spec:
+  rules:
+    - host: artsy-wwwify.artsy.net
```
Comment (izakp) [inline hokusai/staging.yml:143]: @joeyAghion didn't know what DNS name to do for staging?  Perhaps this should also be `artsy.net`?

**Comment by izakp** [comment]
> Sweet. The load-balancer deletion is a separate step, yeah?

Yes will follow up with a PR to switch DNS, then clean up the ELB and Nginx

**Pull request merged**